### PR TITLE
fix test scenario 9 on mobile

### PR DESCRIPTION
**Pull request by gestchild**
⬆️

### CODE REVIEW

File: common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx [hunk 1]
```diff
@@ -180,63 +179,50 @@ const IIIFSearchWithin: FunctionComponent<Props> = ({
...
             return (
               <ListItem key={i}>
-                <NextLink
-                  {...itemLink(
-                    {
-                      canvas: index || 0,
-                    },
-                    'search_within_result'
-                  )}
+                <ListLink
```
Comment (jamesgorrie) [inline common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx:182]: Could we just make it of `type="button"` and we wouldn't need to `preventDefault()`?

File: playwright/test/view-items.test.ts [hunk 2]
```diff
@@ -205,7 +205,9 @@ describe("Scenario 9: A user wants to be able to search inside an item's text",
     }
     await searchWithin('darwin');
     await page.waitForSelector(searchWithinResultsHeader);
-    await page.click(`${searchWithinResultsHeader} + ul li:first-of-type a`);
+    await page.click(
+      `${searchWithinResultsHeader} + ul li:first-of-type button`
```
Comment (jamesgorrie) [inline playwright/test/view-items.test.ts:209]: This still feels like it lacks meaning to me.

**Review by davidpmccormick — APPROVED** [review]

File: common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx [hunk 3]
```diff
@@ -180,63 +179,50 @@ const IIIFSearchWithin: FunctionComponent<Props> = ({
             const matchingCanvas = index && canvases[index];
             return (
               <ListItem key={i}>
-                <NextLink
-                  {...itemLink(
-                    {
-                      canvas: index || 0,
-                    },
-                    'search_within_result'
```
Comment (jamesgorrie) [inline common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx:188]: Can we remove this from the ItemLink too? Or do we use it elsewhere.

**Review by jamesgorrie — APPROVED** [review]
Nice, few nits.

**Pull request merged**